options.html

```diff
@@ -885,5 +927,6 @@ <h3>Images</h3>
 <script src="js/core.js"></script>
 <script src="js/language.js"></script>
 <script src="js/options/options.js"></script>
+<script src="js/content/common.js"></script>
```

Comment:
This was added in order to communicate from the options page to the background.

Reply:
The only reason you need to communicate with the background is to force the contextmenu update?

This would be better triggered from the `chrome.storage.onChanged` event handler in the background. It's possible to change the contents of `SyncedStorage` without visiting the options page.

Reply:
Yeah, I initially used `chrome.storage.onChanged` too, but that was somehow unreliable. It didn't always update.

Reply:
Weird, works fine now. I'll change it. This is much better!